samples/snippets/evaluate_processor_version_sample.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# [START documentai_evaluate_processor_version]
+
+from google.api_core.client_options import ClientOptions
+from google.api_core.exceptions import NotFound
+from google.cloud import documentai_v1beta3 as documentai
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_id = 'YOUR_PROJECT_ID'
+# location = 'YOUR_PROCESSOR_LOCATION' # Format is 'us' or 'eu'
+# processor_id = 'YOUR_PROCESSOR_ID'
+# processor_version_id = 'YOUR_PROCESSOR_VERSION_ID'
+# gcs_input_uri = # Format: gs://bucket/directory/
+
+
+def evaluate_processor_version_sample(
+    project_id: str,
+    location: str,
+    processor_id: str,
+    processor_version_id: str,
+    gcs_input_uri: str,
+):
+    # You must set the api_endpoint if you use a location other than 'us', e.g.:
+    opts = ClientOptions(api_endpoint=f"{location}-documentai.googleapis.com")
+
+    client = documentai.DocumentProcessorServiceClient(client_options=opts)
+
+    # The full resource name of the processor version
+    # e.g.: projects/project_id/locations/location/processors/processor_id/processorVersions/processor_version_id
+    name = client.processor_version_path(
+        project_id, location, processor_id, processor_version_id
+    )
+
+    evaluation_documents = documentai.BatchDocumentsInputConfig(
+        gcs_prefix=documentai.GcsPrefix(gcs_uri_prefix=gcs_input_uri)
+    )
+
+    # NOTE: Alternatively, specify a list of GCS Documents
```

Comment:
I recommend cutting this.

We typically avoid long commented-out blocks of code like this that present alternatives. If this is an important alternative, then it's worth of its own sample.

Reply:
This is an important alternative, but there are multiple samples that can use this same alternative.

Wondering if it's worth the maintenence overhead to create more samples for each?